com.mojang.Minecraft.json

```diff
@@ -77,7 +77,7 @@
         {
           "type": "script",
           "commands": [
-            "/app/extra/minecraft-launcher/minecraft-launcher --workDir=$XDG_DATA_HOME/../.minecraft/"
+            "exec /app/extra/minecraft-launcher/minecraft-launcher $@ --workDir=$XDG_DATA_HOME/../.minecraft/"
```

Comment:
Shouldn't the `$@` variable be placed at the end of the command? Otherwise, lgtm.

Reply:
@nedrichards Feel free to merge it after this is resolved. :)

Reply:
I'm out the country at the moment but I'll fix it up and test it again when I'm home, can only do so much from a web browser.